core/src/main/res/values/strings.xml

```diff
@@ -428,6 +428,8 @@
     <string name="pref_gpodnet_notifications_sum">This setting does not apply to authentication errors.</string>
     <string name="pref_playback_speed_title">Playback Speeds</string>
     <string name="pref_playback_speed_sum">Customize the speeds available for variable speed audio playback</string>
+    <string name="pref_playback_time_respect_speed_title">Time respect playback speed</string>
+    <string name="pref_playback_time_respect_speed_sum">Times and durations respects playback speed</string>
```

Comment:
Times and durations on player screen respect playback speed?

Reply:
done